swap/swap.go

```diff
@@ -54,9 +56,9 @@ type Swap struct {
 	owner               *Owner             // contract access
 	params              *Params            // economic and operational parameters
 	contract            swap.Contract      // reference to the smart contract
-	oracle              PriceOracle        // the oracle providing the ether price for honey
-	paymentThreshold    int64              // balance difference required for sending cheque
-	disconnectThreshold int64              // balance difference required for dropping peer
+	honeyPriceOracle    HoneyOracle        // oracle which resolves the price of honey (in Wei)
+	paymentThreshold    uint64             // amount of debt to a peer at which the node will initiate payment
```

Comment:
I know this is nitpicking, but this is again from one of the two nodes' point of view (debt vs credit). Although I agree that the previous text was not great, I'd change it like with the command line help: "honey amount at which a payment is triggered", likewise the disconnect

Reply:
Shouldn't this be from the point of view from the person who operates the node?

Reply:
Again the same. The threshold does not define if the node operator is indebted or not, it is just a threshold, thus it should be neutral.

Reply:
You did not put back in the comment on this one though and kept it your way. I still disagree with this, I will approve because it's nonsense to drag this any further.

I'd appreciate you'd change it back though before merging.

Reply:
Done